lib/board.rb

```diff
@@ -0,0 +1,74 @@
+class Board
+  attr_accessor :cells, :turn_count
+  #@cells = []
+def initialize
+  @cells = Array.new(9, " ")
+end
+
+def cells
+  @cells
+end
+
+def reset!
+  self.cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+end
+
+def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+end
+
+#def input_to_index(input)
+  #input.to_i - 1
+#end
+#def position(input)
+  #@cells[input_to_index(input)]
+#end
+
+def position(input)
+  position = input.to_i - 1
+  @cells[position]
+end
+
+  def full?
+    @cells.none? do |space|
+      if space == " "
```

Comment:
Avoid returning boolean literals from `if` statements, relying instead on the implicit return value of the boolean expression. This method's content could be `cells.none? {|cell| cell == " "}`